clockwork_web/static/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -204,55 +490,145 @@ msgstr "alloc_tres"
 msgstr "Nombre d'éléments à afficher par page"
 
-#: clockwork_web/templates/settings.html:118
+#: clockwork_web/templates/settings.html:136
 msgid "Dark mode"
 msgstr "Mode sombre"
 
-#: clockwork_web/templates/settings.html:130
-#, fuzzy
+#: clockwork_web/templates/settings.html:148
 msgid "Time formatting"
-msgstr "Plus d'informations"
+msgstr "Formattage de temps"
+
+#: clockwork_web/templates/settings.html:154
+#: clockwork_web/templates/settings.html:156
+msgid "In words (2 months ago)"
+msgstr "Rédigée (Il y a 2 mois)"
+
+#: clockwork_web/templates/settings.html:160
+#: clockwork_web/templates/settings.html:162
+msgid "Unix timestamp (1667503950)"
+msgstr "Timestamp Unix (1667503950)"
 
-#: clockwork_web/templates/settings.html:144
+#: clockwork_web/templates/settings.html:205
 msgid "Visible columns in dashboard table"
-msgstr ""
+msgstr "Colonnes visibles sur le tableau de bord"
 
-#: clockwork_web/templates/settings.html:178
+#: clockwork_web/templates/settings.html:239
 msgid "Visible columns in search table"
-msgstr ""
+msgstr "Colonnes visibles lors de la recherche de jobs"
 
-#: clockwork_web/templates/settings.html:214
+#: clockwork_web/templates/settings.html:275
 msgid "Settings will apply on page reload."
-msgstr ""
+msgstr "Les paramètres seront mis à jour lorsque la page sera rechargée."
 
 #: clockwork_web/templates/single_gpu.html:39
-#: clockwork_web/templates/single_node.html:33
 msgid "key"
 msgstr "clef"
 
 #: clockwork_web/templates/single_gpu.html:40
-#: clockwork_web/templates/single_node.html:34
 msgid "value"
 msgstr "valeur"
 
 #: clockwork_web/templates/single_job.html:26
-#, fuzzy, python-format
+#, python-format
 msgid "Single job %(job_id)s"
-msgstr "job %(job_id)s"
+msgstr "Job %(job_id)s"
+
+#: clockwork_web/templates/single_job.html:32
+msgid "The array below displays the Slurm elements describing the job."
+msgstr ""
+"Le tableau ci-dessous présente les éléments du dictionnaire Slurm "
+"décrivant la job."
+
+#: clockwork_web/templates/single_job.html:36
+#: clockwork_web/templates/single_node.html:35
+msgid "Key"
+msgstr "Clef"
+
+#: clockwork_web/templates/single_job.html:37
+#: clockwork_web/templates/single_node.html:36
+msgid "Value"
+msgstr "Valeur"
 
 #: clockwork_web/templates/single_node.html:24
 #, python-format
-msgid "single node %(node_name)s"
-msgstr "noeud %(node_name)s"
+msgid "Single node %(node_name)s"
+msgstr "Noeud %(node_name)s"
+
+#: clockwork_web/templates/single_node.html:29
+msgid "The array below displays the Slurm elements describing the node."
+msgstr ""
+"Le tableau ci-dessous présente les éléments du dictionnaire Slurm "
+"décrivant le noeud."
+
+#: clockwork_web/templates/single_user.html:40
+msgid "Accounts"
+msgstr "Comptes"
+
+#: clockwork_web/templates/single_user.html:47
+#: clockwork_web/templates/single_user.html:64
+msgid "Used on clusters:"
+msgstr "Utilisé(s) sur les grappes :"
+
+#: clockwork_web/templates/single_user.html:57
+msgid "Director"
```

Comment:
That one is actually incorrect in the English version because it should be "Supervisor", but I think it's been hidden for the time because we don't list that information.